pkgs/os-specific/linux/firmware/fwupdate/default.nix

```diff
@@ -13,7 +20,11 @@ in stdenv.mkDerivation {
     ./do-not-create-sharedstatedir.patch
   ];
 
-  NIX_CFLAGS_COMPILE = "-I${gnu-efi}/include/efi -Wno-error=address-of-packed-member";
+  NIX_CFLAGS_COMPILE = builtins.toString [
```

Comment:
i don't think toString is needed - nix does it for you when you put it into a drv

Reply:
True, I did this for sake of the upcoming structured attrs.